typings/guppy-defs.d.ts

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable camelcase */
+// Code - third_party/static/guppy-b5055b/src/guppy.js
```

Comment:
Wanted to check whether Guppy can also be an auto-upgrade?

Reply:
I don't really know what is the proccess of upgrading the libs in manifest.json. Probably @vojtechjelinek can help here.

Also Guppy's repo has been inactive for about 2 years. And we are using the latest version from that repo I guess.

Reply:
No, it can't since even a minor version needs to be upgraded manually.